examples/launcher-templates/helm/2.4.0-v1/job.tpl

```diff
@@ -88,6 +88,10 @@ spec:
       {{- if or $templateData.pod.serviceAccountName .Job.serviceAccountName }}
       serviceAccountName: {{ .Job.serviceAccountName | default $templateData.pod.serviceAccountName | quote }}
       {{- end }}
+      {{- with $templateData.pod.hostAliases }}
+      hostAliases:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Comment:
FYI @samcofer I noticed while working on this PR that we forgot to add the hostAliases setting to the helm chart's job template version. Sorry about that, I'll try to remember to update those templates going forward